internal-rules/report-message-format.test.ts

```diff
@@ -0,0 +1,80 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils'
+import { stripIndent } from 'common-tags'
+import { fromFixture } from 'eslint-etc'
+import path, { resolve } from 'path'
+import rule, { messageId } from './report-message-format'
+
+function ruleTester() {
+  return new TSESLint.RuleTester({
+    parser: resolve('./node_modules/@typescript-eslint/parser'),
+    parserOptions: {
+      project: resolve('./tests/tsconfig.json'),
+      ecmaVersion: 2020,
+      sourceType: 'module',
+    },
+  })
+}
```

Comment:
Ideally it would be better to import this from an `utils` package.

Reply:
> Oh, you wrote tests for internal rules 😅 I didn't think of that.

This is the price you pay for being a lazy person (me 😄), I'd rather write them than keep generating the js files and manually checking if everything is ok.

> The same counts for the `utils` package.
> I think that if we want that, that we need to move some thing around.
> Is it OK to leave things as is, and reevaluate this later if it's needed?

Yeah, for sure.

Reply:
Oh, I see... yea that makes sense.
Would it be easier to just write these rules in JS?

Reply:
If the answer is yes, we can go ahead and write them.
If you prefer to use TS, we can keep the setup as is with this PR and create a new one and remove the JS files and include a smarter mechanism.
I'm fine either way 👍

While I was writing this, we could also add a third option and remove these internal rules.
And simply have a script that runs on commit, and verifies that the description is how we want it.
This would be identical to how we generate the configs/readme, but it would throw if it detects a mall-formatted description. Maybe this would be the simplest way to create and test it, with the best DX.